BDA_FS19_Projektdokumentation.tex

```diff
@@ -125,9 +125,8 @@
 
 \pagenumbering{Roman}
 
-\renewcommand{\abstractname}{Management Summary}
 \begin{abstract}
-	Abstract
+	Das Ergebnis dieser Arbeit ist eine Machbarkeitsstudie im Bereich der vollautomatischen Suche eines Einzelexemplars in der Kooperativen Speicherbibliothek in Büron. Über 16 Wochen wurden Recherchen durchgeführte, zwei Konzepte ausgearbeitet, über identifizierte Unklarheiten Versuche mit erworbener Hardware durchgeführt und aufgrund der Ergebnisse dieser eine Machbarkeitsstudie geschrieben. In dieser wird beschrieben, wie das Problem von deplatzierten Einzelexemplaren vor der Einlagerung gelöst werden kann, dabei werden Aufwendungen von rund 20'000 Franken für die Entwicklung identifiziert. In dieser Projektdokumentation wird beschrieben wie der Stand der Technik im Bezug auf RFID im Moment aussieht, die gewählte, agile Projektmethode erläutert, die erarbeiteten Konzepte vorgestellt und die Realisierung einer Referenzimplementation beschrieben. Diese Referenzimplementation wurde vor Ort getestet und damit die Machbarkeitsstudie validiert. Weiter wird in der Evaluation und Validation dargestellt, dass die identifizierten Anforderungen erfolgreich erreicht werden konnten und was im Projektverlauf funktioniert hat. Dabei werden sowohl die technischen Hilfsmittel, wie auch die entwickelten Arbeitsprozesse betrachtet. Im Ausblick wird das weitere Vorgehen für das Folgeprojekt erläutert und Empfehlungen für das Folgeteam ausgesprochen. Die Arbeit schliesst mit dem Fazit des Projektteams.
```

Comment:
In dieser wird beschrieben, wie das Problem von deplatzierten Einzelexemplaren vor der Einlagerung gelöst werden **kann. Dabei**